src/main/java/org/moshun/onlinebookstore/service/BookService.java

```diff
@@ -0,0 +1,15 @@
+package org.moshun.onlinebookstore.service;
+
+import java.util.List;
+import org.moshun.onlinebookstore.model.Book;
+
+public interface BookService {
+
+    Book save(Book book);
+
+    List<Book> findAll();
+
+    Book findById(Long id);
+
+    //public Book updateBook(Long id, Book book);
```

Comment:
it's better not to keep unused things in your code. our code is not like notebook, when we want to save unused things for future in order not to forget them - it's carried by git history already
```suggestion
    Book save(Book book);

    List<Book> findAll();

    Book findById(Long id);
```